pkg/cosign/tuf/client.go

```diff
@@ -133,40 +132,63 @@ func (t *TUF) Close() error {
 	return t.local.Close()
 }
 
-func NewFromEnv(ctx context.Context) (*TUF, error) {
-	// Initializes a new TUF object from the local cache or defaults.
-	t, err := newTuf(ctx)
+func initializeTUF(ctx context.Context, embed bool, mirror string, root []byte, forceUpdate bool) (*TUF, error) {
```

Comment:
could you add a comment on the function for what each of the inputs is used for?

Reply:
done